easybuild/easyconfigs/n/NCO/NCO-4.6.8-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,32 @@
+# contributed by Luca Marsella (CSCS)
+easyblock = 'ConfigureMake'
+
+name = 'NCO'
+version = "4.6.8"
+
+homepage = 'http://nco.sourceforge.net/'
+description = """CDO is a collection of command line Operators to manipulate and analyse Climate and NWP model Data."""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'opt': True, 'pic': True}
+
+sources = [SOURCELOWER_TAR_GZ]
+source_urls = ['http://nco.sourceforge.net/src/']
+
+dependencies = [
+    # not using cray-netcdf/4.4.1 because it is compiled with cray-hdf5/1.10
+    # NCO: openmp support built by default, mpi is not yet supported
+    ('ANTLR', '2.7.7', '-python2'),
+    ('netCDF', '4.4.1.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('cray-hdf5/1.10.0', EXTERNAL_MODULE),
```

Comment:
`cray-hdf5/1.10.0.3` is the default on dom

Reply:
Done.